org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/DeviceRequestTest.java

```diff
@@ -219,19 +237,42 @@ public void testInvalidName() throws Exception {
 	}
 
 	@Test
-	public void testMandelModelConfigure() throws Exception {
-		
+	public void testMandelbrotDeviceInfo() throws Exception {
 		DeviceRequest req = new DeviceRequest();
 		req.setDeviceName("mandelbrot");
 		DeviceRequest res = requester.post(req);
-		
+
+		@SuppressWarnings("unchecked")
 		DeviceInformation<MandelbrotModel> info = (DeviceInformation<MandelbrotModel>)res.getDeviceInformation();
-		if (info==null) throw new Exception("There were no devices found and at least the mandelbrot example should have been!");
-		
+		assertNotNull("There were no devices found and at least the mandelbrot example should have been!", info);
+		assertEquals("Example mandelbrot device", info.getDescription());
+		assertEquals(DeviceRole.HARDWARE, info.getDeviceRole());
+		assertNull(info.getHealth()); // TODO what does this attribute mean?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)